documentation/source/simulator_support.rst

```diff
@@ -16,6 +16,23 @@ Accessing bits of a vector doesn't work:
 See ``access_single_bit`` test in :file:`examples/functionality/tests/test_discovery.py`.
 
 
+Verilator
+---------
+
+cocotb supports Verilator 4.020 and above.
+Verilator converts Verilog code to C++ code that is compiled into highly optimized simulation and does not support VHDL.
```

Comment:
"compiled into highly optimized simulation" sounds a bit awkward, but I'm not sure what would be better.

Reply:
"that is compiled, resulting in highly optimized simulation"?